src/main/java/com/cybozu/kintone/client/connection/Connection.java

```diff
@@ -307,7 +268,7 @@ public JsonElement uploadFile(String fileName, InputStream fis) throws KintoneAP
 
             checkStatus(connection, null);
             InputStream inputStream = connection.getInputStream();
-            response = uploadFileWithStream(outputStream, writer, inputStream, fis);
+            response = readStream(inputStream);
```

Comment:
Error may happen in readStream, we need wrap "InputStream inputStream = connection.getInputStream()" in try-with-resources statement.

Reply:
I've updated it in commit [e0880ee](https://github.com/kintone/kintone-java-sdk/pull/38/commits/e0880eeebd87cbc15fa683f400e761a0ca772d44).